src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -172,14 +173,53 @@ public static boolean isPositionInFrame( final int position ) {
         return (((position - 1) % 3) == 0);
     }
 
+    /**
+     * Get the start position of the difference between the reference and alternate alleles in the given {@link VariantContext}.
+     * Assumes the given {@link VariantContext} contains only one alternate allele.
+     * @param variant {@link VariantContext} in which to determine the start of the different bases.
+     * @return The start position of the difference between the reference and alternate alleles in the given {@link VariantContext}.
+     */
+    public static int getIndelAdjustedAlleleChangeStartPosition(final VariantContext variant) {
+        return getIndelAdjustedAlleleChangeStartPosition(variant, variant.getAlternateAllele(0));
+    }
+
+    /**
+     * Get the start position of the difference between the reference and alternate alleles in the given {@link VariantContext}.
+     * @param variant {@link VariantContext} in which to determine the start of the different bases.
+     * @param altAllele The alternate {@link Allele} against which to check the reference allele in {@code variant}.
+     * @return The start position of the difference between the reference and alternate alleles in the given {@link VariantContext}.
+     */
+    public static int getIndelAdjustedAlleleChangeStartPosition(final VariantContext variant, final Allele altAllele) {
+
+        // If the variant is an indel, we need to check only the bases that are added/deleted for overlap.
+        // The convention for alleles in Funcotator is to preserve a leading base for an indel, so we just need
+        // to create a new variant that has its start position shifted by the leading base.
+        // NOTE: because there could be degenerate VCF files that have more than one leading base overlapping, we need
+        //       to detect how many leading bases there are that overlap, rather than assuming there is only one.
+        final int varStart;
+        if ( GATKProtectedVariantContextUtils.typeOfVariant(variant.getReference(), altAllele).equals(VariantContext.Type.INDEL) ) {
```

Comment:
I think you should also check that it is not a complex indel (`GATKProtectedVariantContextUtils.isComplexIndel(...)`), unless that is addressed upstream...

Reply:
Yup.  Sounds good.  Fixed!